x-pack/plugins/siem/public/components/open_timeline/index.test.tsx

```diff
@@ -229,76 +282,97 @@ describe('StatefulOpenTimeline', () => {
   });
 
   describe('#onSelectionChange', () => {
-    test('it updates the selection state when timelines are selected', () => {
-      const wrapper = mountWithIntl(
-        <StatefulOpenTimeline
-          deleteTimelines={jest.fn()}
-          defaultPageSize={DEFAULT_SEARCH_RESULTS_PER_PAGE}
-          openTimeline={jest.fn()}
-          searchResults={mockResults}
-          title={title}
-        />
+    test('it updates the selection state when timelines are selected', async () => {
+      const wrapper = mount(
+        <TestProviderWithoutDragAndDrop>
+          <MockedProvider mocks={mockOpenTimelineQueryResults} addTypename={false}>
+            <StatefulOpenTimeline
+              apolloClient={apolloClient}
+              isModal={false}
+              defaultPageSize={DEFAULT_SEARCH_RESULTS_PER_PAGE}
+              title={title}
+            />
+          </MockedProvider>
+        </TestProviderWithoutDragAndDrop>
       );
 
+      await wait();
+
       wrapper
         .find('.euiCheckbox__input')
         .first()
         .simulate('change', { target: { checked: true } });
+
       wrapper.update();
 
-      expect(get('selectedItems', wrapper.state()).length).toEqual(9); // 9 selectable timelines are shown on the first page of data
+      expect(get('selectedItems', getStateChildComponent(wrapper).state).length).toEqual(13); // 9 selectable timelines are shown on the first page of data
```

Comment:
The comment is now saying `9`, but the code says `13`. I would update the comment.